packages/fx-core/tests/common/local/localEnvManager.test.ts

```diff
@@ -75,7 +75,6 @@ describe("LocalEnvManager", () => {
 
       chai.assert.isDefined(projectSettings);
       chai.assert.isUndefined(projectSettings.appName);
-      chai.assert.isUndefined(projectSettings.projectId);
```

Comment:
Why removing this?
If projectId has default value, validate the expected value instead of removing the assert.

Reply:
So better to just put `chai.assert.isDefined(projectSettings.projectId);` here.

Reply:
I will change this to isDefined